addons/portal/views/portal_templates.xml

```diff
@@ -99,6 +100,28 @@
         <button type="button" class="close" data-dismiss="alert" aria-label="Close"> &#215; </button>
     </template>
 
+    <template id="portal_side_bar" name="My Account">
+        <div id="o_my_sidebar" class="pt-3 pt-lg-0 col-12 col-md col-lg-4 col-xl-3 o_my_sidebar">
```

Comment:
yeah there is another template called portal_sidebar. I will rename this one portal_my_account_sidebar instead to avoid confusion. The class o_my_sidebar was written for this use case, so I will rename it o_portal_my_account_sidebar